packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -18,7 +29,7 @@ const items: Item[] = [
   { value: 5, label: 'entry out of viewport' },
 ];
 
-const AutocompleteDefaultStory = ({ items }: { items: Item[] }) => {
+export const Basic = (args: AutocompleteProps<{}>) => {
```

Comment:
Can I pass the empty `{}` there? Would it be correct? 🤔

Reply:
You can but it's not very useful then. Maybe it's fine for the story.